usl_models/tests/model_test.py

```diff
@@ -218,3 +219,35 @@ def test_early_stopping():
     history = model.train([fake_data], early_stopping=1)
     # Check whether the model history indicates early stopping.
     assert len(history[0].history["val_loss"]) < params.epochs
+
+
+def test_model_checkpoint():
+    """Tests saving and loading a model checkpoint."""
+    batch_size = 16
+    height, width = 100, 100
+    params = model_params.test_model_params
+
+    model = flood_model.FloodModel(params, spatial_dims=(height, width))
+
+    storm_duration = 4
+    fake_input = fake_flood_input_batch(
+        batch_size,
+        height=height,
+        width=width,
+        storm_duration=storm_duration,
+        include_labels=True,
+    )
+    fake_data = flood_model.FloodModelData(storm_duration=storm_duration, **fake_input)
+    model.train([fake_data])
+
+    model_path = "/tmp/test_model.keras"
```

Comment:
we'll want to use a tempfile to make sure this gets deleted, and also for comparability with our windows friends who don't have a /tmp folder

https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile

```python
with tempfile.NamedTemporaryFile() as tmp:
  model.save_model(tmp.name)
  ...
```

Reply:
Done.